.vscode/launch.json

```diff
@@ -9,18 +9,19 @@
       "args": [
         "--colors",
         "--timeout=0",
-        "./built/test"
+        "./../../../test/"
       ],
-      "cwd": "${workspaceRoot}",
+      "cwd": "${workspaceRoot}/bazel-out/local-fastbuild/bin/test/golden_test.runfiles/io_angular_tsickle",
```

Comment:
Isn't this the same as bazel-bin/ without some intervening dirs?

Reply:
I'm using this path because this lines up better with the source mappings we produce.